src/other/Paginator.js

```diff
@@ -23,9 +23,38 @@
     Paginator.prototype.publish("numItems", 10, "number", "Pagination total number of items",null,{tags:["Private"]});
     Paginator.prototype.publish("pageNumber", 1, "number", "Pagination set or get the page number",null,{tags:["Private"]});
 
+
+    Paginator.prototype.testData = function() {
+        return this;
+    };
+
     Paginator.prototype.enter = function (domNode, element) {
         HTMLWidget.prototype.enter.apply(this, arguments);
-        this.paginator = element.append("ul").attr("class","paginator pagination pagination-sm");
+        var context = this;
+
+        this.container = element.append("div");
+        this.paginator = this.container.append("ul").attr("class","paginator pagination pagination-sm");
+        this.side = this.container.append("div").attr("class","paginator pagination side");
+
+        this.side.append("span")
+            .classed("side", true)
+            .text("Page ")
+        ;
+
+        this.side.append("input")
+            .attr("type","text")
```

Comment:
If you change this to "number" it would allow the user to use the up/down arrow keys to switch between pages (when the input has focus).